config-default.yaml

```diff
@@ -76,4 +76,12 @@ disabledTokens:
       symbol: SNX-old
       description: This token is disabled for trading and deposit. SNX will be deprecated and replaced by another token at the end of July.  Go to https://www.synthetix.io for more information
       reason: DEPRECATED
+      
+    - address: '0x45804880De22913dAFE09f4980848ECE6EcbAf78'
+      name: Paxos Gold
+      symbol: PAXG
+      description: This token is disabled for trading and deposit.
+      reason: DISABLED
```

Comment:
should we not append DISABLED to paxg name?

Reply:
Should we? @anxolin

Reply:
I would say is better like it is now imo

![image](https://user-images.githubusercontent.com/2352112/86246396-361ea080-bbab-11ea-8114-ecd11b521552.png)


you can see the warning and the faded icon. I think is good!